api/src/main/resources/application-local.properties

```diff
@@ -1,15 +1,21 @@
-auth.idam.client.baseUrl=http://localhost:5000
-enable.fee.loader=false
+spring.application.name=fees-register-api
+spring.jackson.serialization.fail-on-unwrapped-type-identifiers=false
+
+spring.main.allow-bean-definition-overriding=true
+
 spring.datasource.url=jdbc:postgresql://localhost:5432/fees_register
 spring.datasource.username=fees_register
 spring.datasource.password=fees_register
-server.port=8091
-server.shutdown=graceful
-spring.lifecycle.timeout-per-shutdown-phase=30s
-spring.jpa.properties.hibernate.show_sql=true
-spring.jpa.properties.hibernate.format_sql=true
-idam.api.url=${IDAM_API_URL:http://localhost:5000}
-spring.security.oauth2.client.provider.oidc.issuer-uri = http://localhost:5000/o
-oidc.audience-list = FEES_CLIENT_ID
-oidc.issuer = ${OIDC_ISSUER:http://fr-am:8080/openam/oauth2/hmcts}
-trusted.user.roles=freg-editor,freg-approver,freg-admin
+spring.datasource.driver=org.postgresql.Driver
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+
+spring.liquibase.enabled=${SPRING_LIQUIBASE_ENABLED:true}
+
+fees.loader.json=feeLoader.json
+
+auth.idam.client.baseUrl=http://localhost:23443
+auth.provider.service.client.baseUrl=http://localhost:23443
+
+management.endpoints.web.base-path=/
+management.endpoints.web.exposure.include=health,info
+management.endpoint.health.show-details=always
```

Comment:
Should changes to the local setup file be included in a PR? Guessing this is related to your issue with port 5000 but merging in these changes might cause issues for others who have it setup previously